src/overlays/actors/ovl_En_Encount2/z_en_encount2.h

```diff
@@ -32,7 +32,7 @@ typedef struct EnEncount2 {
     /* 0x01D0 */ EnEncount2Particle particles[200];
 } EnEncount2; // size = 0x2A70
 
-#define GET_ENCOUNT2_SWITCH_FLAG(this)((s16) (this->dynaActor.actor.params & 0x7F))
+#define GET_ENCOUNT2_SWITCH_FLAG(this)((s16) (this->dyna.actor.params & 0x7F))
```

Comment:
This should probably be updated to use the base Actor* pointer to be consistent.

Reply:
ah i could do that, i just had noticed that this actor used `dynaActor` instead of `dyna` while fixing a warning in it, but i can make that change